.gitignore

```diff
@@ -237,3 +237,6 @@ fabric.properties
 
 # Android studio 3.1+ serialized cache file
 .idea/caches/build_file_checksums.ser
+
+# custom
+/fake_generator_images/
```

Comment:
What do you think of the idea, to create a directory which is completely ignored. Then the different generatores could add subdirectories where the put there data by default. This way we do not need to extend the gitignore by every generator/processor. A name for that directory could be "output".

Reply:
well i dont intend to push all images, that future generators will create, therefore we should ignore it no? i agree with the 2nd point tho, we could push one directory that contains all future outputs directories, i can change that.

Reply:
> we could push one directory that contains all future outputs directories

Yeah just create one, add it to .gitignore and let your generator output its images to an subdirectory.